samples/stock/plotoptions/series-cumulativestart/demo.js

```diff
@@ -0,0 +1,40 @@
+const day = 1000 * 60 * 60 * 24;
+
+// Create the chart
+Highcharts.stockChart('container', {
+
+    title: {
+        text: 'CumulativeStart'
+    },
+
+    plotOptions: {
+        series: {
+            cumulative: true,
+            pointStart: Date.UTC(2021, 0, 1),
+            pointInterval: day
+        }
+    },
+
+    rangeSelector: {
+        enabled: false
+    },
+
+    tooltip: {
+        pointFormat: '<span style="color:{series.color}">{series.name}</span>' +
+            ': <b>{point.y}</b> ({point.cumulativeSum})<br/>',
+        changeDecimals: 2,
+        valueDecimals: 2
+    },
+
+    xAxis: {
+        minRange: day * 3,
+        max: Date.UTC(2021, 0, 6)
```

Comment:
Instead of `max` we should use `min`, to present the second point as the initial. Right now, we just show the first point, so it's unclear what's going on

Similar to the test